config/fe-code-convention.md

```
@@ -0,0 +1,210 @@
+| 1 | 레이아웃 | display, visibility, overflow, float, clear, position, top, right, bottom, left, z-index |
+| 2 | BOX | width, height, margin, padding, border |
+| 3 | 배경 | background |
+| 4 | 폰트 | font, color, letter-spacing, text-align, text-decoration, text-indent, vertical-align, white-space |
+| 5 | 기타 | 위에 언급되지 않은 나머지 속성들로 폰트 관련 속성 이후에 선언하며, 기타 속성 내의 선언 순서는 무관함 |
+
+## 변수
+
+변수명 (축약, Container, Wrapper, Box)
+
+- boolean값
+    - 수동태(접미사 ed)로 하기!
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)